build/goreleaser/linux/rhel_9_amd64.yml

```diff
@@ -6,7 +6,7 @@
       - linux-ctl-amd64
       - linux-service-amd64
     package_name: newrelic-infra
-    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el9.{{ .Arch }}"
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el9.{{ .Arch }}{{ .Env.FIPS }}"
```

Comment:
Same, this should be :

`"newrelic-infra{{ .Env.FIPS }}-{{ .Env.TAG }}-1.el9.{{ .Arch }}"`